app/src/main/java/com/github/orkest/ui/feed/FeedActivity.kt

```diff
@@ -335,16 +329,21 @@ private fun LikeButton(viewModel: PostViewModel, post: Post) {
                 .height(20.dp)
                 .width(20.dp)
                 .clickable {
-                    viewModel
-                        .updatePostLikes(post, !isPostLiked.value)
-                        .thenApply {
-                            //Updating the isPostLiked value accordingly
-                            isPostLiked.value = !isPostLiked.value
-                            //Updating the value of getNbLikes only after the updatePostLikes future has been completed
-                            viewModel
-                                .getNbLikes(post)
-                                .thenApply { nbLikes.value = it }
-                        }
+                    if(FireStoreDatabaseAPI.isOnline(context)){
+                        viewModel
+                            .updatePostLikes(post, !isPostLiked.value)
+                            .thenApply {
+                                //Updating the isPostLiked value accordingly
+                                isPostLiked.value = !isPostLiked.value
+                                //Updating the value of getNbLikes only after the updatePostLikes future has been completed
+                                viewModel
+                                    .getNbLikes(post)
+                                    .thenApply { nbLikes.value = it }
+                            }
+                    }
+                    else{
+                        Toast.makeText(context, "No internet connection. Unable to like.", Toast.LENGTH_LONG).show()
```

Comment:
Did you find a way to test these Toast messages, I've been looking for a way to do it but it didn't work yet

Reply:
No I didn't find a way that was the problem